website/src/docs/hotchocolate/performance/persisted-queries.md

```diff
@@ -2,12 +2,239 @@
 title: "Persisted queries"
 ---
 
-Hi,
+This guide will walk you through how standard persisted queries works and how you can set them up with the Hot Chocolate GraphQL server.
 
-We're currently working on the version 11 documentation. Probably right now at this very moment. However, this is an open-source project, and we need any help we can get! You can jump in at any time and help us improve the documentation for hundreds or even thousands of developers!
+# What they are
 
-In case you might need help, check out our slack channel and get immediate help from the core contributors or the community itself.
+Allows you to pre-register all required queries/mutations from your client on your GraphQl server.
 
-Sorry for any inconvenience, and thank you for being patient!
+## Why to use
 
-The ChilliCream Team
+There are two main reasons why to use persisted queries:
+
+- Performance - Payload contains only Hash and variables. This will reduce the size of your client application since queries can be removed from the client code at the build time.
+
+- Security - By using persisted queries, the server will accept only known queries / mutations and refuse all others that are not part of persisted "Allowed-List". Useful mainly for public APIs. 
+
+## How it works
+
+This can be done by extracting the queries from your client application at build time and putting them to server query storage. Persisted client output contains queries / mutations that your client application needs to run. Each extracted query has a specific Hash (identifier) and the client uses it with Variables to query data from your server. There is no more query string in the body of your request. The server obtains only concrete identifier + variables and searches for them in AllowedList. If there is a match, the request is directly executed without the need for query parsing else the particular error message is returned to the request initiator. 
+
+## Limitations
+
+If your API is used by multiple consumers out of your environment with custom requirements for query / mutation, then this is not for your use case and you may have a look at [automatic persisted queries](automatic-persisted-queries), which probably fits better your scenario. 
+
+## How they are stored
+
+Persisted queries are stored close to your server either in the file system or in a Redis cache this helps to reduce request sizes.
+
+[HC provides various places to store this queries](https://www.nuget.org/packages?q=Hotchocolate.PersistedQueries) :
+
+Nuget namespaces:
+- HotChocolate.PersistedQueries.Redis 
+- HotChocolate.PersistedQueries.FileSystem  
+- HotChocolate.PersistedQueries.InMemory
+
+# Setup
+In the following tutorial, we will walk you through creating a Hot Chocolate GraphQL server and configuring it to support standard persisted queries.
+
+## Step 1: Create a GraphQL server project
```

Comment:
I would leave this part out and just start where you add the `HotChocolate.PersistedQueries.FileSystem` package.